src/map/clif.cpp

```diff
@@ -11045,6 +11045,9 @@ void clif_parse_LoadEndAck(int fd,struct map_session_data *sd)
 			//Login Event
 			npc_script_event(sd, NPCE_LOGIN);
 		}
+
+		// Set facing direction before battle_config.spawn_direction
+		sd->ud.dir = sd->status.dir;
```

Comment:
Why not unit_setdir?

Reply:
Also the question is if you should actually do that only if the config is enabled?
Because otherwise server and client direction are out of sync or not?